recipes/omniscidb-cuda/recipe/build.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Make sure we are in the right place
+cd $SRC_DIR
+
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
+
+# Make sure -fPIC is not in CXXFLAGS (that some conda packages may
```

Comment:
Why not?

Reply:
omniscidb server generates machine code from LLVM IR during runtime and with `-fPIC` in CXXFLAGS the server crashes when executing the generated code. Why it crashes is hard to diagnose but removing `-fPIC` helped.